src/common/components/select-repository-list/index.js

```diff
@@ -196,9 +184,9 @@ export class SelectRepositoryListComponent extends Component {
               { selectedRepositories.length } selected
             </strong>
             {' '}
-            (<a href="#" onClick={this.onHelpClick.bind(this)}>
+            (<Button appearance={ 'link' } onClick={this.onHelpClick.bind(this)} aria-label={ this.state.showMissingReposInfo ? 'Click to close the missing / private repos dialog' : 'Click to open the missing / private repos dialog' }>
```

Comment:
I would say "Anything missing?" doesn't really gives decent context for the page or area of content it's in. What is missing exactly? If anything is missing what am I expecting to see when I click this? 

@matthewpaulthomas can you supply possibly an improvement to this, thus I would remove the ```aria-label``` because currently as it stands a user whom is vision impaired or requires any other form of extra context wouldn't know what is missing. 

Regarding to the link comments, unless the ```<a>``` element links to an ID or new page with in the site / app, a button element is far more applicable and semantic for the page.